source/renderer/app/components/wallet/WalletRestoreDialog.js

```diff
@@ -313,6 +322,12 @@ export default class WalletRestoreDialog extends Component<Props, State> {
 
         walletData.type = this.state.walletType;
 
+        if (walletData.type === WALLET_RESTORE_TYPES.YOROI_REGULAR) {
+          walletData.type = WALLET_RESTORE_TYPES.REGULAR;
+        } else if (walletData.type === WALLET_RESTORE_TYPES.YOROI_LEGACY) {
```

Comment:
@DeeJayElly this is no good. We should not mask the `YOROI_LEGACY` as a normal `LEGACY` wallet. This info is crucial to know what type of a wallet we are restoring in the Store.

Reply:
Fixed